docs/07-physics.md

```diff
@@ -0,0 +1,339 @@
+---
+title: Physics
+path: /docs/physics
+---
+
+Excalibur comes built in with two physics systems. The first system is [[CollisionResolutionStrategy.Box|Box physics]], and is a
+simple axis-aligned way of doing basic collision detection for non-rotated rectangular areas, defined by an actor's
+[[BoundingBox|bounding box]].
+
+### Physics hierarchy
+
+Excalibur physics are organized into a hierarchy, each has a specific single role in the collision system.
+
+```
+Actor (game entity)
+  -> Body (transform information)
+     -> Collider (collision related information)
+        -> Shape (geometry for collision)
+```
+
+For example:
+
+```typescript
+const actor = new ex.Actor({
+  pos: new ex.Vector(100, 100),
+  body: new ex.Body({
+    vel: new ex.Vector(20, 0), // velocity
+    acc: new ex.Vector(0, 100), // acceleration
+    collider: new ex.Collider({
+      mass: 100, // mass of 100
+      type: ex.CollisionType.Active, // active collision type
+      shape: ex.Shape.Circle(50) // circle geometry of radius 50
+    })
+  })
+});
+```
+
+### Actor/Body
+
+Actor's have position, velocity, and acceleration in physical space, all of these positional physical attributes are contained inside the [[Body]]. Only 1 actor can be associated with a [[Body]].
+
+**[[Body]]** is the container for all transform related information for physics and any associated colliders.
+
+This looks like this:
+
+```typescript
+const actor = new ex.Actor({
+    // actor's position is stored on a default body
+    pos: new ex.Vector(40, 40);
+});
+
+// actor position is stored on the body, actor.pos is a convenience
+actor.pos === actor.body.pos
+
+// actor velocity is stored on the body, actor.vel is a convenience
+actor.vel === actor.body.vel
+
+// actor acceleration is stored on the body, actor.acc is a convenience
+actor.acc === actor.body.acc
+```
+
+### Collider
+
+[[Body]]'s have a default box collider that is derived from the width and height of the [[Actor]] associated. Only 1 [[Collider]] can be associated with a [[Body]], and by extension an [[Actor]]. (Collision events are re-emitted onto [[Actor]])
+
+**[[Collider]]** is the container of all collision related information, collision type, collision events, mass, inertia, friction, bounciness, shape, etc.
+
+### Shape
```

Comment:
All shapes could do with an example picture displaying the shape with example coordinates.